classes/class2/README.md

```diff
@@ -1 +1,303 @@
-#Class 2
+#JavaScript
+
+## JavaScript is Everywhere
+
+### Atwood's Law
+	
+>Any application that can be written in JavaScript, will eventually be written in JavaScript.
+
+[Jeff Atwood](http://blog.codinghorror.com/the-principle-of-least-power/)
+
+## The Language
+
+Luckily, you're probably already familiar with a language similar to JavaScript. Things like variable assignment,function definition, and control flow aren't much different from Python. We'll get into the differences later.
+
+Type `node` in your command line to bring up the Node JavaScript REPL (read-evaluate-print-loop).
```

Comment:
I think we can briefly mention in class that the dev console is available, but leave it at that until we actually get to the client-side class where we will introduce it.